src/applications/vaos/appointment-list/components/AppointmentsPageV2/Grid.jsx

```diff
@@ -0,0 +1,210 @@
+      )}`,
+      appointmentType: getVideoAppointmentLocationText(appointment),
+      icon: '',
+    };
+  }
+  if (isPhone) {
+    return {
+      appointmentDetails: 'VA Appointment',
+      appointmentType: 'Phone call',
+      icon: 'fas fa-phone vads-u-margin-right--1',
+    };
+  }
+  if (isInPersonVAAppointment()) {
+    const { name } = getTypeOfCareById(serviceType) || {};
+    return {
+      appointmentDetails: `${name} with ${getPractitionerName(appointment)}`,
+      appointmentType: `In person at ${appointment.vaos.facilityData?.name}`,
+      icon: '',
+    };
+  }
+  return {
+    appointmentDetails: '',
+    appointmentType: '',
+    icon: '',
+  };
+}
+
+function isCanceled(appointment) {
+  return appointment.status === APPOINTMENT_STATUS.cancelled;
+}
+
+function getLabelText(appointment) {
+  const appointmentDate = getAppointmentDate(appointment);
+
+  return `Details for ${
+    isCanceled(appointment) ? 'canceled ' : ''
+  }appointment on ${appointmentDate.format('dddd, MMMM D h:mm a')}`;
+}
+
+export default function Grid({
+  appointment,
+  link,
+  handleClick,
+  handleKeyDown,
+}) {
+  const { isCommunityCare, isVideo } = appointment.vaos;
+  const canceled = isCanceled(appointment);
+  const appointmentDate = getAppointmentDate(appointment);
+  const { abbreviation, description } = getAppointmentTimezone(appointment);
+  const label = getLabelText(canceled);
+  const { appointmentDetails, appointmentType } = getGridData(appointment);
+  const styles = {
+    canceled: {
+      textDecoration: canceled ? 'line-through' : 'none',
+    },
+  };
+
+  return (
+    <>
+      {/* Disabling for now since add role=button and tab=0 fails another accessiblity check: */}
+      {/* Nested interactive controls are not announced by screen readers */}
+      {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+      <div
+        className={classNames(
+          // 'vads-u-padding--2',
+          'vads-u-display--flex',
+          'vads-u-align-items--left',
+          'vads-u-flex-direction--column',
+          'medium-screen:vads-u-flex-direction--row',
+          'medium-screen:vads-u-align-items--center',
+        )}
+        onClick={handleClick()}
+        onKeyDown={handleKeyDown()}
+      >
+        <div className="vads-u-flex--auto vads-u-margin-y--neg0p5">
+          <Label label="Canceled" />
+          <div style={{ width: '33px' }}>
+            <h4 className="vads-u-margin-y--0 vads-u-font-size--h3 vads-u-padding-right--1 vads-u-padding-y--2">
+              {appointmentDate.format('D')}
+            </h4>
+            <span className="sr-only"> {description}</span>
+          </div>
+        </div>
+        <div className="vads-u-flex--auto vads-u-padding-y--2">
+          <div style={{ width: '69px' }}>{appointmentDate.format('ddd')}</div>
+        </div>
+        <div className="vads-u-flex--auto  vads-u-padding-y--2 vads-u-border-bottom--1px vads-u-border-color--gray-lightest">
```

Comment:
just making the border color slightly darker
```suggestion
        <div className="vads-u-flex--auto  vads-u-padding-y--2 vads-u-border-bottom--1px vads-u-border-color--gray-lighter">
```